site/app/views/submission/HomeworkView.php

```diff
@@ -151,82 +159,82 @@ public function showGradeable($gradeable, $days_late) {
-            $return .= <<<HTML
+            if ($gradeable->hasAssignmentMessage()) {
+                $return .= <<<HTML
         <p class='green-message'>{$gradeable->getAssignmentMessage()}</p>
 HTML;
-        }
-        $return .= <<<HTML
+            }
+            $return .= <<<HTML
     </div>
 HTML;
-        if($gradeable->useSvnCheckout()) {
-            $return .= <<<HTML
+            if($gradeable->useSvnCheckout()) {
+                $return .= <<<HTML
     <input type="submit" id="submit" class="btn btn-primary" value="Grade SVN" />
 HTML;
-        }
-        else {
-            $return .= <<<HTML
+            }
+            else {
+                $return .= <<<HTML
     <div id="upload-boxes" style="display:table; border-spacing: 5px; width:100%">
 HTML;
 
-            for ($i = 0; $i < $gradeable->getNumTextBoxes(); $i++) {
-
-                $image_width = $image_height = 0;
+                for ($i = 0; $i < $gradeable->getNumTextBoxes(); $i++) {
 
-                if (isset($gradeable->getTextboxes()[$i]['images']) && $gradeable->getTextboxes()[$i]['images'] != ""){
-                    $tester = $gradeable->getTextboxes()[$i]['images'];
-                }
-                else{
-                    $tester = array();
-                }
+                    $image_width = $image_height = 0;
 
-                //
-                foreach((array)$tester as $currImage){
-                    $currImageName = $currImage["image_name"];
-                    $imgPath = $this->core->getConfig()->getCoursePath() . "/test_input/" . $gradeable->getName() . "/".$currImageName;
-                    $content_type = FileUtils::getContentType($imgPath);
-                    if (substr($content_type, 0, 5) === "image") {
-                       // Read image path, convert to base64 encoding
-                       $textBoxImageData = base64_encode(file_get_contents($imgPath));
-                       // Format the image SRC:  data:{mime};base64,{data};
-                       $textBoximagesrc = 'data: '.mime_content_type($imgPath).';charset=utf-8;base64,'.$textBoxImageData;
-                       // insert the sample image data
-
-                        if(isset($currImage['image_height']) && (int)$currImage['image_height'] > 0){
-                            $image_height = $currImage['image_height'];
-                        }
+                    if (isset($gradeable->getTextboxes()[$i]['images']) && $gradeable->getTextboxes()[$i]['images'] != ""){
+                        $tester = $gradeable->getTextboxes()[$i]['images'];
+                    }
+                    else{
+                        $tester = array();
+                    }
 
-                        if(isset($currImage['image_width']) && (int)$currImage['image_width'] > 0){
-                            $image_width = $currImage['image_width'];
+                    //
+                    foreach((array)$tester as $currImage){
```

Comment:
I am not sure, that part of the code I just indented in because of it being wrapped in a new 'if' condition.